internal/engine/pod.go

```diff
@@ -46,6 +46,12 @@ func handlePodChangeAction(ctx context.Context, state *store.EngineState, pod *v
 		return
 	}
 
+	fwdsValid := portForwardsAreValid(manifest, *podInfo)
```

Comment:
it seems very bad for a reducer to be emitting warnings, because this means that the warning will be emitted whenever there's a state change. can you leave a todo for me about this?

Reply:
sure -- or we could move that warning to `PortForwardController.OnChange`?